src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,17 @@
+import '../styles/ArticleCard.css'
+
+export const ArticleCard = ({ article }) => {
+    return (
+        <div className="article-card"
+            style={{
```

Comment:
I get what you're going for here, be careful that the white text doesn't get lost in the graphic